foodsaving/groups/templates/user_inactive_in_group.subject.jinja2

```diff
@@ -0,0 +1 @@
+[{{ group_name }}] {% trans %}is missing you!{% endtrans %}
```

Comment:
Still not sure about the subject, it seems a bit weird to have brackets in a sentence

Reply:
Agree

Reply:
... and updated